system/datatypes/peer_reviewed_article.php

```diff
@@ -1202,13 +1224,21 @@ protected function createEditForm(array $values = []) {
 			</div>
 EOF;
 
+		$src = array_key_exists(self::FIELD_NAME_FEATURED_IMAGE, $values) && is_string($values[self::FIELD_NAME_FEATURED_IMAGE]) && file_exists($values[self::FIELD_NAME_FEATURED_IMAGE]) ? $values[self::FIELD_NAME_FEATURED_IMAGE] : null;
+		$src = str_replace(HyphaImage::ROOT_PATH, '', $src);
```

Comment:
This would probably be more readable as an `if` statement.

Reply:
If the `ROOT_PATH` is not saved, this whole part can be removed.